pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperkits/JuniperKitCreationTest.java

```diff
@@ -1,43 +1,66 @@
 package org.broadinstitute.dsm.juniperkits;
 
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
 import java.util.List;
 import java.util.Random;
 
 import com.google.gson.Gson;
-import com.typesafe.config.Config;
-import org.broadinstitute.dsm.DSMServer;
-import org.broadinstitute.dsm.TestHelper;
+import org.broadinstitute.dsm.DbTxnBaseTest;
+import org.broadinstitute.dsm.db.DDPInstance;
 import org.broadinstitute.dsm.db.KitRequestShipping;
 import org.broadinstitute.dsm.model.nonpepperkit.JuniperKitRequest;
 import org.broadinstitute.dsm.model.nonpepperkit.KitResponse;
+import org.broadinstitute.dsm.model.nonpepperkit.KitResponseError;
 import org.broadinstitute.dsm.model.nonpepperkit.NonPepperKitCreationService;
-import org.broadinstitute.dsm.statics.ApplicationConfigConstants;
+import org.broadinstitute.dsm.util.EasyPostUtil;
+import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
-import org.junit.Ignore;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+
+/**
+ * Tests the NonPepperKitCreationService class
+ * To run this class use the following VM variables
+ * -ea -Dconfig.file=[path to /pepper-apis/output-build-config/testing-inmemorydb.conf]
+ *  */
+
+@RunWith(MockitoJUnitRunner.class)
+public class JuniperKitCreationTest extends DbTxnBaseTest {
 
-public class JuniperKitCreationTest {
     NonPepperKitCreationService nonPepperKitCreationService = new NonPepperKitCreationService();
-    private Config cfg;
+    final String instanceGuid = "Juniper-mock-guid";
```

Comment:
Nit: Typically we would use the term 'test' instead of 'mock', since the latter has a very specific meaning.